nemo/core/classes/modelPT.py

```diff
@@ -90,13 +91,14 @@ def save_to(self, save_path: str):
             save_path: Path to .nemo file where model instance should be saved
         """
         with tempfile.TemporaryDirectory() as tmpdir:
-            config_yaml = path.join(tmpdir, _MODEL_CONFIG_YAML)
-            model_weights = path.join(tmpdir, _MODEL_WEIGHTS)
-            self.to_config_file(path2yaml_file=config_yaml)
-            trainer = self._trainer if self._trainer else Trainer()
-            if trainer.model != self:
-                trainer.model = self
-            trainer.save_checkpoint(filepath=model_weights, weights_only=True)
+            # Prepare filenames.
+            config_yaml_file = path.join(tmpdir, _MODEL_CONFIG_YAML)
+            model_weights_file = path.join(tmpdir, _MODEL_WEIGHTS)
+            # Save the configuration.
+            self.to_config_file(path2yaml_file=config_yaml_file)
+            # Save the weights.
+            save(self.state_dict(), model_weights_file)
```

Comment:
Please use the explicit torch.save() here to have semantic understanding of which "save" we are actually calling.